packages/yoastseo/src/values/Paper.js

```diff
@@ -258,6 +259,22 @@ Paper.prototype.getCustomData = function() {
 	return this._attributes.customData;
 };
 
+/**
+ * Checks whether a text title is available.
+ * @returns {boolean} Returns true if the Paper has a text title.
+ */
+Paper.prototype.hasTextTitle = function() {
+	return this._attributes.textTitle !== "";
```

Comment:
Is this a safe check? If, for whatever reason textTitle is undefined, this function will erroneously return True.
Maybe add a check with `isNil()` for lowdash? Note: check if `isNil` is imported.

```suggestion
	return this._attributes.textTitle !== "" && ! isNil(this._attributes.textTitle);
```